api/app/signals/settings/development.py

```diff
@@ -8,9 +8,9 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 SIGNALS_AUTHZ = {
-    'JWKS_URL': "http://dex:5556/keys",
-    'ALWAYS_OK': False,
-    'USER_ID_FIELD': 'email'
+    'JWKS_URL': os.getenv('JWKS_URL', 'http://dex:5556/keys'),
+    'ALWAYS_OK': os.getenv('ALWAYS_OK', 'false') == 'true',
```

Comment:
Maybe cast this to lowercase, in that case it is also possible to set `True` and `TRUE`. Do we want to support `1` and `0` values to be considered True/False?

Reply:
I copied the approach from https://github.com/Amsterdam/signals/blob/6d9785014f98bc6f71fcdb1d7d71c846ee45e86d/api/app/signals/settings/base.py#L159.

Think we should have a consistent way of handling booleans through environment variables.

Reply:
@bartjkdp which I have changed recently into `if os.getenv('SWIFT_ENABLED', False) in [True, 1, '1', 'True', 'true']:`

Reply:
Changed this to `in ['True', 'true', '1']`.